test/runtests.jl

```diff
@@ -1,4 +1,15 @@
-using FillArrays, LinearAlgebra, PDMats, SparseArrays, StaticArrays, ReverseDiff, Random, Base64, Test, Statistics
+using Base64
```

Comment:
Don't make unneccesary style-related changes like this in an unrelated PR.

I personally don't like having packages on different lines like this. It just takes up more space in the file for no reason. And I see no benefit in making these types of changes.

Reply:
I'll revert this change, but this stylistic change is not for no reason. Firstly, having an alphabetically sorted list makes it easy to check which dependencies are being imported, and secondly, it makes git diffs easier to read if a new package is added or if one is removed. A few extra lines in an already 2000+ line file probably doesn't matter very much. Ultimately, it's subjective, of course, but I would urge you to consider such changes in the future